org.siril.Siril.json

```
@@ -12,13 +12,16 @@
     "finish-args": [
         "--share=ipc",
         "--share=network",
-        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
         "--filesystem=host",
+        /* For spawning OS-native processes */
+        /*"--talk-name=org.freedesktop.Flatpak", */
         /* Needed for gvfs to work */
         "--talk-name=org.gtk.vfs.*",
         "--filesystem=xdg-run/gvfs",
         "--filesystem=xdg-run/gvfsd",
-        "--share=network"
+        "--unset-env=LD_PRELOAD"
```

```suggestion
        "--unset-env=LD_PRELOAD"
```

I don't see why this is needed. LD_PRELOAD is always unset by flatpak.